nbconvert/exporters/exporter.py

```diff
@@ -160,11 +160,8 @@ def from_filename(self, filename, resources=None, **kw):
             Ignored
 
         """
-        # Convert full filename string to unicode
-        # In python 2.7.x if filename comes as unicode string,
-        # just skip converting it.
-        if isinstance(filename, str):
-           filename = py3compat.str_to_unicode(filename)
+        if not isinstance(filename, str):
```

Comment:
Probably better to just delete this check and add a type annotation to the function:

```python
def from_filename(self, filename: str, resources: Optional[dict] = None, **kw)
```

Reply:
Not that you have the type hint above I don't think you need this line at all

Reply:
whoops! just fixed